sqlflow/client.py

```diff
@@ -180,4 +207,6 @@ def _decode_any(cls, any_message):
                 any_message.Unpack(timestamp_message)
                 return timestamp_message.ToDatetime()
             raise TypeError("Unsupported type {}".format(any_message))
-
+    
+    def _enable_argo_mode(self):
```

Comment:
Can we determine the mode based on the return value of `Run` RPC call? For example, if the return value contains a job message, the client should execute fetch. This ease the deployment by avoiding configuring environment variable `SQLFLOW_ARGO_MODE`.